cargo-ledger/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "cargo-ledger"
+version = "1.2.0"
```

Comment:
Bump the version ?

Reply:
Then we should update the version in the app-builder docker images

Reply:
OK I will bump the version to 1.2.1

Reply:
I will update the version in the app builder image in https://github.com/LedgerHQ/ledger-app-builder/pull/91